Makefile

```diff
@@ -236,6 +241,16 @@ $(RELEASE_DIR_BIN)/%:
 ###############################################################################
 # Utilities
 ###############################################################################
+helm: bin/helm
+HELM_DISTRO=linux-amd64
```

Comment:
Might want to auto-detect this, or use a container?

Reply:
I think this needs to stay hardcoded because it is going to get executed inside the jekyll container which is linux based, even when run on a mac. Right?

Reply:
In that case, why make it a variable?

Reply:
touche